src/Kconfig.authentication

```diff
@@ -28,4 +28,23 @@ config GOLIOTH_HARDCODED_CA_CRT_PATH
     help
       Path to a certificate that will be used to authenticate the server
 
+config GOLIOTH_SECONDARY_CA_CRT
+    bool "Include a secondary CA certificate"
+    default y
+    help
+      Optionally include a second CA certificate. We strongly recommend
+      you enable this option and use the default secondary certificate.
+      This will ensure the greatest compatability with the Golioth
+      servers going forward.
```

Comment:
I'm open to making this language stronger. I.e. removing the bit about "optionally" and saying that turning off this config or changing the default voids any guarantees that Golioth makes about future compatibility.